lib/msal-common/src/response/AuthenticationResult.ts

```diff
@@ -32,10 +33,11 @@ export type AuthenticationResult = {
     accessToken: string;
     fromCache: boolean;
     expiresOn: Date | null;
+    extExpiresOn?: Date;
+    refreshOn: Date | null;
```

Comment:
What is the difference between

`refreshOn: Date | null;` 
and
`refreshOn?: Date;`